examples/complete/main.tf

```diff
@@ -1,37 +1,46 @@
-###########################################
+##################################################################################
 ### Supporting resources
-###########################################
+### NOTE: Run this example in SQS Server-side encryption supported regions like
+### US East (N. Virginia), US East (Ohio), and US West (Oregon) regions
+##################################################################################
+data "aws_kms_alias" "aws_default" {
```

Comment:
Does this still apply? weren't you going to use `kms_master_key_id = "alias/aws/sqs"` for the examples? And wouldn't it make sense to set that as the default?

For this example that would be enough.